software/messaging/src/main/java/org/apache/brooklyn/entity/zookeeper/ZooKeeperNode.java

```diff
@@ -46,7 +46,7 @@
 
     @SetFromFlag("downloadUrl")
     AttributeSensorAndConfigKey<String, String> DOWNLOAD_URL = ConfigKeys.newSensorAndConfigKeyWithDefault(SoftwareProcess.DOWNLOAD_URL,
-            "http://apache.fastbull.org/zookeeper/zookeeper-${version}/zookeeper-${version}.tar.gz");
+            "http://apache.org/dist/zookeeper/zookeeper-${version}/zookeeper-${version}.tar.gz");
```

Comment:
From `http://apache.org/dist/`:

`Please do not download from apache.org! If you are currently at apache.org and would like to browse, please visit a nearby mirror site instead.`

Would be fine though if we add the binary to cloudsoft's mirror - it's queried first, right?